### PR TITLE
Address 5 issues including: Add examples of escaping syntax and add text about non-linguistic sorting

### PR DESCRIPTION
**Pull request by aphillips**
Addresses #3

Addresses #62

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/65.html" title="Last updated on May 10, 2022, 4:25 PM UTC (0b0a812)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/65/43f96d7...aphillips:0b0a812.html" title="Last updated on May 10, 2022, 4:25 PM UTC (0b0a812)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1388,6 +1390,64 @@ <h3>Designing character escapes</h3>
...
+				<tr>
+					<th width="20%">Type</th>
+					<th>Application</th>
+					<th width="20%">Example</th>
+					<th>Description</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Hex NCRs</td>
```
Comment (xfq) [inline index.html]:
```suggestion
					<td>Hex <abbr title="numeric character references">NCRs</abbr></td>
```

File: index.html [hunk 2]
```diff
@@ -1388,6 +1390,64 @@ <h3>Designing character escapes</h3>
...
+					<td>UTR#18</td>
+					<td>Perl</td>
+					<td><code>\x{1F63D}</code></td>
+					<td>Variable-width; uses <code>x</code> instead of the more common <code>u</code></td>
+				</tr>
+				<tr>
+					<td>C (UTF-32)</td>
+					<td>C</td>
+					<td><code>\U0001f63d</code></td>
+					<td>Fixed-width; encodes UTF-32 code units; most often used to encode supplementary characters together with  lowercase <code>u</code> escapes used for BMP characters, e.g. <kbd>\u00c0 \U001f63d \u12fe</kbd></td>
```
Comment (xfq) [inline index.html]:
```suggestion
					<td>Fixed-width; encodes UTF-32 code units; most often used to encode supplementary characters together with  lowercase <code>u</code> escapes used for <abbr title="Basic Multilingual Plane">BMP</abbr> characters, e.g. <kbd>\u00c0 \U001f63d \u12fe</kbd></td>
```

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html]: Good callouts. See if my edits in ff085fe address these.

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html]: See ff085fe

**Review by r12a — APPROVED** [review]
\o/ great job

**Pull request merged**